packaging/suse/bin/portusctl

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+bundle="/srv/Portus/vendor/bundle/ruby/2.1.0/bin/bundler.ruby2.1"
+export GEM_PATH=/srv/Portus/vendor/bundle/ruby/2.1.0/
```

Comment:
This is not possible, that's a bash script. That's a dirty trick to run portusctl's ruby code using the gems that we bundle into Portus. Otherwise we would have to make rubygem-thor a dependency of Portus (despite this gem being already inside of the internal bundle)